### PR TITLE
Update basic to not be over-subscribed.

### LINKED ISSUE #1424: [RM Ch04] Remove oversubscription from basic
Issue #973 documented the intent to both park compute intensive flavour and to remove oversubscription from the Basic flavour.

The Parking of compute-intensive was done, but the removal of oversubscription was not.  This issue addresses the removal of over-subscription by setting CPU allocation ratio to 1:1 for Basic flavour.

### PR DESCRIPTION
**Pull request by kedmison**
Fixes #1424.  

Issue #973 documented the intent to both park compute intensive flavour and to supplement it by removing oversubscription from the Basic.  The work that was done in #973 only changed the compute intensive flavour but not the oversubscription.

### CODE REVIEW

**Comment by rabi-abdel** [comment]
@kedmison would you like to keep this on draft? or move it to go though the review process.

**Comment by kedmison** [comment]
I have just had an idea.  let me see if it will work.

File: doc/ref_model/chapters/chapter05.md [hunk 1]
```diff
@@ -137,15 +137,15 @@ This section will detail NFVI Software Profiles and associated configurations fo
 
 | .conf            | Feature                | Type   | Basic | Network Intensive |
 |------------------|------------------------|--------|-------|-------------------|
-| nfvi.com.cfg.001 | CPU allocation ratio   | value  | 4:1   | 1:1               |
+| nfvi.com.cfg.001 | CPU allocation ratio   | value  | 1:1   | 1:1               |
```
Comment (tomkivlin) [inline doc/ref_model/chapters/chapter05.md:140]: Suggest adding a note below the table stating that this is to allow for future performance benchmarking, allowing operators to modify for actual deployments if they are willing to accept the risk of an impact on performance.

File: doc/ref_model/chapters/chapter04.md [hunk 2]
```diff
@@ -478,7 +478,7 @@ n100, n200, n300, n400, n500, n600 | N | Y | N
 | `e.cap.016`<br />(FPGA/other Acceleration H/W)                        | Yes (if offered)         | No                       | || | `e.cap.017`<br />(Monitoring of L2-7 data)                  | No                       | Yes                      | Exposed monitoring capabilities as per [**Table 4-3**](#Table4-3)   |
 | `i.cap.014`<br />(CPU cores consumed by the NFVI on the worker nodes) | any                      | any                      | |
 | `i.cap.015`<br />(Memory consumed by NFVI on the worker nodes)        | any                      | any                      | |
-| `i.cap.016`<br />(CPU allocation ratio)                     | 4:1                      | 1:1                      | Internal SLA capabilities as per [**Table 4-6**.](#Table4-6)        |
+| `i.cap.016`<br />(CPU allocation ratio)                     | 1:1                      | 1:1                      | Internal SLA capabilities as per [**Table 4-6**.](#Table4-6)        |
```
Comment (tomkivlin) [inline doc/ref_model/chapters/chapter04.md:481]: Suggest adding a note below the table stating that this is to allow for future performance benchmarking, allowing operators to modify for actual deployments if they are willing to accept the risk of an impact on performance.

File: doc/ref_model/chapters/chapter05.md (hunk 1 above)
Comment (kedmison) [reply doc/ref_model/chapters/chapter05.md]: Sounds good.

**Pull request merged**